app/assets/stylesheets/admin/loans/_charts.scss

```diff
@@ -1,3 +1,23 @@
-.chart {
-  text-align: center;
+.breakeven-chart {
+  margin: 2em .5em 1em;
+  page-break-before: always;
+
+  .section-title, .chart-title {
+    display: block !important;
```

Comment:
Elizabeth to fix. Don't use .section-title and .chart-title. Try to use CSS instead to make this scope specific enough that h3 and h4 can be used.